lucetc/tests/wasm/duplicate_imports.wat

```diff
@@ -0,0 +1,34 @@
+(module
```

Comment:
Could we add a version of this test to the `lucet-wasi` suite, so that we can observe it working correctly at runtime?